JASP-Desktop/html/js/jaspwidgets.js

```diff
@@ -559,7 +566,10 @@ JASPWidgets.NoteBox = JASPWidgets.View.extend({
 		});
 
 		quillEditorElement.addEventListener('focusout', (event) => {
-			self.setQuillToolbarVisibility('none');
+
+			if (!self.hovered) {
+				self.setQuillToolbarVisibility('none');
+			}
```

Comment:
I'm not sure when this situation happens (out of focus, but still hovering), but it does not seem to me the right way to handle this: if there is a focus out and it is still hovered, then the visibility is not set to false, but then when it becomes not hovered anymore, the visibility will still not be set to none. So the Quill Editor stays displayed even it is has no focus and is not hovered.

Reply:
Image resizer module adds an overlay around the image. When the image is clicked the editor loses focus and into the style div element (overlay). 

I was trying to fix the below behaviour. Any suggestions how to approach this?
![jasp-resize-3](https://user-images.githubusercontent.com/6959308/91039295-9ca0c780-e60c-11ea-9cd8-75973957a922.gif)

Reply:
@akashrajkn : which Qt version do you use? Because on Qt 5.14, I get this error when I try to select an image: 
`An unhandled type 'QList<QUrl>' was provided in FilePickerController::accepted(QVariant)`
When I google, I see that this error is solved in 5.15: https://bugreports.qt.io/browse/QTWB-51
So I'm afraid we have to wait the release afterwards to be able to add this feature.